Scan/STLcd/lcd_scan.c

```diff
@@ -342,7 +342,34 @@ inline void LCD_setup()
 // LCD State processing loop
 inline uint8_t LCD_scan()
 {
-	return 0;
+  static uint16_t hold_color[3];
```

Comment:
Please use tabs. My general rule, is just make code look the same as everything around it.

- No egyptian braces :P
- Functions and if statement parenthesis should have a trailing/leading space
```c
if ( !is_capslock )
{
}
```

Logic wise, everything looks great :D